pkg/network/protocols/http/ebpf_javatls.go

```diff
@@ -78,7 +106,51 @@ func (p *JavaTLSProgram) GetAllUndefinedProbes() (probeList []manager.ProbeIdent
 	return
 }
 
+// isAttachmentAllowed will return true if the pid can be attach
+// The filter is based on the process command line matching javaAgentAllowRegex and javaAgentBlockRegex regex
+// javaAgentAllowRegex has an higher priority
+//
+// # In case of only one regex (allow or block) is set, the regex will be evaluated as exclusive filter
+// /                 match  | not match
+// allowRegex only    true  | false
+// blockRegex only    false | true
+func isAttachmentAllowed(pid uint32) bool {
+	// filter is disabled (default configuration)
+	if javaAgentAllowRegex == nil && javaAgentBlockRegex == nil {
+		return true
+	}
+
```

Comment:
```suggestion
func isAttachmentAllowed(pid uint32) bool {
	// filter is disabled (default configuration)
    javaAgentAllowRegexIsEmpty = javaAgentAllowRegex == nil
    javaAgentBlockRegexIsEmpty = javaAgentBlockRegex == nil
	if javaAgentAllowRegexIsEmpty && javaAgentBlockRegexIsEmpty {
		return true
	}
```

Reply:
nit improvement for readability along with https://github.com/DataDog/datadog-agent/pull/15601#discussion_r1113807500